internal/constants/constants.go

```diff
@@ -231,9 +234,15 @@ b92PDCpM7FZAINQF88s1TZS/HmRXYk62UJ4eqPduvUnJmXhNikhLbMi6fw==
 `
 )
 
-// VersionInfo returns the version of a binary.
-func VersionInfo() string {
-	return versionInfo
+// BinaryVersion returns the version of this Binary.
+func BinaryVersion() semver.Semver {
+	version, err := semver.New(versionInfo)
+	if err != nil {
+		// This is not user input, unrecoverable, should never happen.
+		panic(fmt.Sprintf("parsing embedded CLI version: %s", err))
```

Comment:
```suggestion
		panic(fmt.Sprintf("parsing embedded version information: %s", err))
```
This function is also called by the bootstrapper, and other Constellation binaries.